src/main/java/mate/model/Driver.java

```diff
@@ -54,4 +62,20 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, name, licenseNumber);
     }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
```

Comment:
Maybe we need to add login field into equals and hashCode? And I think it would be better to place getters and setters over the equals and hashCode

Reply:
agree, fixed